task-sdk/pyproject.toml

```diff
@@ -134,6 +134,7 @@ exclude_also = [
 [dependency-groups]
 codegen = [
     "datamodel-code-generator[http]==0.28.2",
+    "openapi-spec-validator>=0.7.1",
```

Comment:
This idea cribbed shamelessly from @abhishekbhakat via #49608

Reply:
Good observation!